tests/controllers/np-creation/chainsaw-test.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: np-creation
+spec:
+  timeouts:
+    assert: 10s
+  steps:
+  - try:
+    # first operation: create the config map
+    - apply:
```

Comment:
Favour [create](https://kyverno.github.io/chainsaw/latest/operations/create/) over `apply` operation to ensure a deterministic environment by creating resources from scratch. `apply` might lead to unintended changes due to merging with existing configurations.

Reply:
It is creating issue with delete operation, as whether the resource should be created at once or not is non-deterministic with delete operation so that's why apply is being used to make sure that the resource exists even after the deletion test executes, for other tests.

Reply:
In this test, we're verifying whether an NP is created or not as part of SI and SIB creation, so how is it causing issues with the `delete` operation? Don't all tests start from a clean state?

Reply:
I am also having the same doubt I had pushed the message in the slack channel of chainsaw waiting for the reply from the team